cmd/api/src/go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/zenazn/goji v1.0.1
 	go.uber.org/mock v0.2.0
 	golang.org/x/crypto v0.18.0
+	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc
```

Comment:
Did we intentionally want this dependency added?

Reply:
Removed